app/__init__.py

```diff
@@ -1,10 +1,21 @@
 
 from flask import Flask
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy import create_engine
 
 def create_app():
-    app = Flask(__name__)
-    app.config.from_envvar('SCHEDULR_SETTINGS')
-    from app.modules import test_module, auth
-    app.register_blueprint(test_module.bp)
-    app.register_blueprint(auth.bp, url_prefix='/auth')
-    return app
+    # Setup flask app
+    flapp = Flask(__name__)
+    flapp.config.from_envvar('SCHEDULR_SETTINGS')
+
+    # Setup SQLAlchemy engine & Session class
+    flapp.db_engine = create_engine(flapp.config['DB_STRING'])
+    flapp.db_Session = sessionmaker(bind=flapp.db_engine)
+
+    # Setup blueprints
+    from app.modules import test_module, auth, setup
```

Comment:
Unrelated note but why does this get imported here instead of the top of the file?